index.html

```diff
@@ -425,11 +425,12 @@ <h2>LDP-NRs</h2>
         <h2>HTTP PUT</h2>
         <p>
           When accepting a <code>PUT</code> request against an existing resource, an HTTP <code>Link: rel="type"</code>
-          header MAY be included. If the type in the Link header is a subtype of a current type of the resource, and
-          has an interaction model assigned to it by [[!LDP]], then the resource MUST be assigned the new type and the
-          interaction model of the resource MUST be changed to the interaction model assigned to the new type by
-          [[!LDP]]. If that type is a value in the LDP namespace and is not either a current type of the resource or a
-          subtype of a current type of the resource, the request MUST be rejected with a 409 (Conflict) response.
+          header MAY be included. If the type in the Link header is a subtype of a current type of the resource, is in
+          the [[!LDP]] namespace, and has an interaction model assigned to it by [[!LDP]], then the resource MUST be
+          assigned the new type and the interaction model of the resource MUST be changed to the interaction model
+          assigned to the new type by [[!LDP]]. If that type is a value in the [[!LDP]] namespace and is neither a
```

Comment:
_"If *the* type ..."_ -- seems odd to have *that* all of a sudden when the rest says *the*